tempest-internal/src/main/kotlin/app/cash/tempest/internal/Schema.kt

```diff
@@ -293,7 +293,9 @@ data class ItemType(
       for (expectedAttribute in expectedRawItemAttributes) {
         require(rawItemType.propertyNames.contains(expectedAttribute)) {
           "Expect $expectedAttribute, required by $itemType, to be declared in " +
-            "${rawItemType.type}. But found ${rawItemType.propertyNames}. Use @Transient to exclude it."
+            "${rawItemType.type}. But found ${rawItemType.propertyNames}. Use @Transient to exclude it." +
```

Comment:
Nit: missing space after each `.` in all the lines that were changed.

Reply:
Good catch. Fixed!